pkg/skaffold/update/update_test.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package update
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/config"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+)
+
+func TestIsUpdateCheckEnabledByEnvOrConfig(t *testing.T) {
+	tests := []struct {
+		description string
+		envVariable string
+		cfg         *config.GlobalConfig
+		readErr     error
+		expected    bool
+	}{
+		{
+			description: "nothing is set opt-in behavior",
+			expected:    true,
+		}, {
```

Comment:
this testcase formatting is pretty but it's inconsistent from the other test cases: 
```suggestion
		},
		{
```